espnet/nets/pytorch_backend/transformer/subsampling.py

```diff
@@ -17,21 +17,26 @@ class Conv2dSubsampling(torch.nn.Module):
     :param int idim: input dim
```

Comment:
@pengchengguo, could you give @unilight your comments?
This is what we wanted to do?

Reply:
Yes. Hi @unilight, we also need a custom positional encoding type for conformer model. To keep consistent with transformer's code, we want to add a parameter `pos_enc` to class `Conv2dSubsampling`. By default, `pos_enc=None` and we use `PositionalEncoding`. For specific scenarios, we will pass a `torch.nn.Module` to select different types, such as `ScaledPositionalEncoding` or `RelPositoinalEncoding`.  I think your implementation is same as we discussed.

Reply:
Thank you for your explanation, @pengchengguo ! If my implementation is correct, I will keep it this way.